pkg/cmd/cli/root.go

```diff
@@ -0,0 +1,304 @@
+package cli
+
+import (
+	"github.com/redhat-ai-dev/rhdh-ai-catalog-cli/pkg/cmd/cli/backstage"
+	"github.com/redhat-ai-dev/rhdh-ai-catalog-cli/pkg/cmd/cli/kserve"
+	"github.com/redhat-ai-dev/rhdh-ai-catalog-cli/pkg/cmd/cli/kubeflowmodelregistry"
+	"github.com/redhat-ai-dev/rhdh-ai-catalog-cli/pkg/config"
+	"github.com/redhat-ai-dev/rhdh-ai-catalog-cli/pkg/util"
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const (
+	bkstgAIExample = `
+# Access a supported backend for AI Model metadata and generate Backstage Catalog Entity YAML for that metadata
+$ %s new-model <kserve|kubeflow> <owner> <lifecycle> <args...>
```

Comment:
A bit, but I went with `new-model` given that OCP users are very used to `oc new-app...` and the like.

That said I can always add it as another alias minimally.  I'll do that since that is trivial.  Long term, let's wait until some external feedback while still in alpha phase for broader name changes.

Reply:
But sure I'll add more elaboration / attempt to make clear.